releases/releases.go

```diff
@@ -45,6 +45,37 @@ func Get(c *deis.Client, appID string, version int) (api.Release, error) {
 	return release, reqErr
 }
 
+// Deploy a release
```

Comment:
If you could add a little more detail here, that would be great because it used to generate the [go documentation](https://godoc.org/github.com/deis/controller-sdk-go/releases)